deploy/alerts/alerts.yaml

```diff
@@ -0,0 +1,136 @@
+
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    prometheus: stf-default
+    role: alert-rules
+  name: prometheus-alarm-rules
+  namespace: service-telemetry
+spec:
+  groups:
+    - name: ./openstack.rules
+      rules:
+        - expr: 'rate(collectd_disk_disk_io_time_io_time_total[5m])'
+          record: 'job:iotime:rate_5m'
+        - expr: 'stddev_over_time(job:iotime:rate_5m[1h])'
+          record: 'job:iotime:rate_5m:stddev_over_time_1h'
+        - expr: 'avg_over_time(job:iotime:rate_5m[1h])'
+          record: 'job:iotime:rate_5m:avg_over_time_1h'
+        - alert: iotime warn
+          expr: >-
+            (abs(job:iotime:rate_5m - job:iotime:rate_5m:avg_over_time_1h) / job:iotime:rate_5m:stddev_over_time_1h) > 3
```

Comment:
Purely curiosity, but how did you decide on 3 StdDevs rather than 2 for a warning? I've actually never modeled this on a real workload to see what's best - is there a best practice you read somewhere, or is this an informed guess? Looking at the use of 3 & 6 StdDevs I'm going to guess "six sigma"?

Reply:
sorry, I didn't see this earlier.

I switched to 3 and 6 stddevs since I was under the impression, with 1 and 2 I was getting too many false alarms.